src/datepicker/datepicker-month-view.ts

```diff
@@ -17,6 +17,9 @@ import {DayTemplateContext} from './datepicker-day-template-context';
       height: 100%;
       cursor: pointer;
     }
+    .day:not(.disabled):hover {
+      background-color: rgba(0, 0, 0, 0.075);
```

Comment:
We can't do this as it would kill theming... We need to find a BS class instead to apply on mouseover...